workers/ohsome_quality_analyst/definitions.py

```diff
@@ -145,6 +145,30 @@ def load_topic_definitions() -> dict[str, TopicDefinition]:
     return topics
 
 
+def get_topic_definitions(project: str = None) -> dict[str, TopicDefinition]:
+    topics = load_topic_definitions()
+    if project:
+        return {k: v for k, v in topics.items() if v.project == project}
+    else:
+        return topics
+
+
+def get_indicator_definitions(project: str = None) -> dict[str, IndicatorMetadata]:
+    indicators = load_metadata("indicators")
+    if project:
+        return {k: v for k, v in indicators.items() if v.project == project}
+    else:
+        return indicators
+
+
+def get_report_definitions(project: str = None) -> dict[str, ReportMetadata]:
+    reports = load_metadata("reports")
+    if project:
+        return {k: v for k, v in reports.items() if v.project == project}
+    else:
+        return reports
+
+
 def get_topic_definition(topic_key: str) -> TopicDefinition:
```

Comment:
```suggestion
def get_topic_definition(topic_key: str) -> dict[str, TopicDefinition]:
```
To be consistent with the other functions, this should also be a dict with the key as key and the rest as values. 
Currently, the response of `/metadata/topics/building_count` is missing the key. 

This comment is also valid for /metadata/indicators/key and /metadata/reports/key

Reply:
fixed in 866e9919d4e3c9677869abdb62c1b5d8eab7b4f4